package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "twilio-cli",
-  "version": "5.4.1",
```

Comment:
Why is this version change required?

Reply:
I also didn't understand why the version change so I double checked. The only explanation I can think of is that the `package-lock.json` wasn't updated and committed on the last [release](https://github.com/twilio/twilio-cli/compare/5.8.0...5.8.1)

All I did was run `npm install ngrok@v5.0.0-beta.1 --save-optional`. You can check that these are the changes that take place on the `package-lock.json` by running that command on the `main` branch.